code/core/src/components/components/tooltip/WithTooltip.tsx

```diff
@@ -133,7 +133,7 @@ const WithTooltipPure = ({
       {/* @ts-expect-error (non strict) */}
       {typeof tooltip === 'function' ? tooltip({ onHide: () => onVisibleChange(false) }) : tooltip}
     </Tooltip>
-  );
+  ) : null;
```

Comment:
This is an optimization that makes a lot of sense to me...
Why call a function which we will not use the returned JSX of?